package.json

```diff
@@ -32,6 +32,7 @@
     "dev": "start-storybook -p 6006",
     "create-package": "./tools/create-package.js",
     "create-package:components": "./tools/create-package-components.js",
+    "generate-icons": "./tools/generate-icons.js",
```

Comment:
Should we run this during the build?

Reply:
+1

Reply:
@mshwery @akleiner2 run during build and  don't push generated files?